lisp/ess-mode.el

```diff
@@ -311,18 +311,10 @@ indentation style. See `ess-style-alist' for predefined styles."
   ;; completion
   (add-hook 'comint-dynamic-complete-functions 'ess-complete-filename nil 'local)
   (delq t comint-dynamic-complete-functions)
-  (set (make-local-variable 'comint-completion-addsuffix)
-       (cons "/" ""))
-  (add-hook 'hack-local-variables-hook #'ess--set-style-in-buffer nil t)
+  (setq-local comint-completion-addsuffix (cons "/" ""))
+  (add-hook 'hack-local-variables-hook 'ess-set-style nil t)
```

Comment:
ess-set-style should be sharp-quoted.

Reply:
It doesn't matter in this case and I think both we and core emacs mostly uses simple quote. I find it cleaner this way. Probably just a habit though.

Reply:
I am leaving this with simple quote to match the majority style. Judging from the info file for #' it's not useful for symbols in general. Remembering to add #' in elisp is just an annoyance not existing in other lisps.